packages/oscal-react-library/src/components/HoverablePopover.tsx

```diff
@@ -26,6 +26,29 @@ export const HoverablePopover: React.FC<HoverablePopoverProps> = ({ popoverConte
 
   const open = Boolean(anchorEl);
 
+  const popover = popoverContent ? (
+    <Popover
+      open={open}
+      sx={{
+        pointerEvents: "none",
+        maxWidth: "30em",
+      }}
+      anchorOrigin={{
+        vertical: "bottom",
+        horizontal: "left",
+      }}
+      anchorEl={anchorEl}
+      transformOrigin={{
+        vertical: "top",
+        horizontal: "left",
+      }}
+      onClose={handlePopoverClose}
+      disableRestoreFocus
+    >
+      <Box sx={{ padding: "1em" }}>{popoverContent}</Box>
+    </Popover>
+  ) : null;
```

Comment:
I was thinking about cases in which  `popoverContent` is not defined and was unsure of the best functionality. I would think that the popover should not appear if there is no content to display.

Reply:
Then maybe to simplify the whole thing...

```ts
if (!popoverContent) {
  return (<>{children}</>);
}
```

Reply:
But also it's remarkably unlikely `popoverContent` will be falsey since it's like to be React component with a falsey child. So in any case, _this_ doesn't feel like the right place to do the check?

Reply:
Yeah that is the issue I am running into. I had originally thought it would be falsy if the `MarkupProse` returned `null`. I may just add a check like:

```TypeScript
<HoverablePopover popoverContent={item && <MarkupProse>{item}</MarkupProse>} />
```